appliancevm/ansible/appliancevm.py

```
@@ -113,15 +113,16 @@
 copy(copy_arg, host_post_info)
 
 # name: install virtualenv
-virtual_env_status = check_and_install_virtual_env(virtualenv_version, trusted_host, pip_url, host_post_info)
-if virtual_env_status is False:
-    command = "rm -rf %s && rm -rf %s" % (virtenv_path, appliancevm_root)
+py_version = get_virtualenv_python_version(virtenv_path, host_post_info)
```

Ruff (0.15.2)

[error] `get_virtualenv_python_version` may be undefined, or defined from star imports

(F405)